src/main/java/org/lk/controller/GreetingRequest.java

```diff
@@ -0,0 +1,23 @@
+package org.lk.controller;
+
+public class GreetingRequest {
```

Comment:
Can we name it like `AuthorizaitonRequest`?

Reply:
Yes, we can use name like AuthorizaitonRequest